cpp/include/kvikio/parallel_operation.hpp

```diff
@@ -156,7 +156,7 @@ std::future<std::size_t> parallel_io(F op,
                                       decltype(devPtr_offset)>);
 
```

Comment:
Since we only use page size here, should we just define it here

```suggestion
  static auto const page_size = static_cast<std::size_t>(sysconf(_SC_PAGESIZE));

```

Would also need to add this to the file header

```cpp
#include <unistd.h>
```

Reply:
I'd disagree.

First, the meaning of `static` will change in this context, and will give `page_size` an internal linkage. There's no benefit of doing so. Imagine, in the future, we have an inline function using `page_size` and that inline function is used in more than one TUs, this will be a violation of ODR and cause undefined behavior.

Second, making `page_size` a static global variable will result in every TU that includes `parallel_operation.hpp` calling `sysconf()` before the main function. With this, we lose the benefit of lazy evaluation from the static variable `page_size` in the function `get_page_size()`.

In short, I think the current implementation in this PR is optimal.